private/model/api/eventstream.go

```diff
@@ -974,6 +974,16 @@ func (c *loopReader) Read(p []byte) (int, error) {
 			t.Fatalf("expect no error got, %v", err)
 		}
 
+		// Assert calling Err before close does not close the stream.
+		resp.{{ $esMemberName }}.Err()
+		select {
+		case _, ok := <-resp.{{ $esMemberName }}.Events():
+			if !ok {
+				t.Fatalf("expect stream not to be closed, but was")
+			}
+		default:
```

Comment:
Do we need to generate a default empty case?

Reply:
This default case is triggered when the `case` channel would block. The chan read case will trigger if the `Events` channel is closed. This check is asserting that the channel returned by the `Events` method is blocking.